docs/src/pages/components/dialogs/ConfirmationDialog.js

```diff
@@ -61,8 +61,6 @@ function ConfirmationDialogRaw(props) {
 
   return (
     <Dialog
```

Comment:
Shouldn't we add an `onClose` handler here, something like:

```jsx
onClose={(event, reason) => {
   if (reason !== 'backdropClick') {
     onClose(event, reason);
   }
}}
```

Reply:
From what I understand we aren't interested in the backdrop nor escape events, so the demo didn't use onClose.